src/components/Cart.js

```diff
@@ -1,3 +1,49 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { HeaderNavBar } from "./customComponents/HeaderNavBar";
+import { deleteFromCart, fetchCartItems } from "./utils/requestUtils/CartRequestUtils";
+
 export const Cart = () => {
-    return <div>Cart</div>
+    const [cartProducts, setCartProducts] = useState([]);
+    useEffect(() => {
+        (async () => {
+        const products = await fetchCartItems();
+        console.log(`Cart: ${JSON.stringify(products.cart)}`)
```

Comment:
remove the console.logs

Reply:
Added it for debugging purposes. Will remove them at the end.